src/base.js

```diff
@@ -25,6 +27,47 @@ const case_ids = [];
 const copiedTestResults = [];
 const expectedFailures = {};
 
+class ZealTestRail extends TestRail {
+    constructor(config) {
+        super(config);
+        this.host = config.host;
+        this.username = config.username;
+        this.password = config.password;
+    }
+
+    async addAttachmentToResult(attachmentPath, resultId) {
```

Comment:
@gevorg-vardanyan-im  jan, 
Could you clarify the reason for implementing this method from scratch instead of using the existing `addAttachmentToResult` method from the `@dlenroc/testrail` npm package (https://www.npmjs.com/package/@dlenroc/testrail)?

Reply:
The reason is that the addAttachmentToResult method from the @dlenroc/testrail package does not support attaching video files.
Creating a custom implementation was necessary to avoid this issue.